{{cookiecutter.repo_name}}/docs/source/conf.py

```diff
@@ -12,47 +12,48 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 
-# Incase the project was not installed
+# In case the project was not installed
 import os
 import sys
-sys.path.insert(0, os.path.abspath('../..'))
-import {{cookiecutter.repo_name}}  # noqa
+sys.path.insert(0, os.path.abspath("../.."))
+import {{cookiecutter.package_name}}  # noqa
 
 
 
 # -- Project information -----------------------------------------------------
 
-project = '{{cookiecutter.project_name}}'
+project = "{{cookiecutter.project_name}}"
 copyright = (
-    "{% now 'utc', '%Y' %}, {{cookiecutter.author_name}}. "
+    "{% now "utc", "%Y" %}, {{cookiecutter.author_name}}. "
     "Project structure based on the "
     "MDAnalysis Cookiecutter version {{cookiecutter._mda_cc_version}}"
 )
-author = '{{cookiecutter.author_name}}'
+author = "{{cookiecutter.author_name}}"
 
 # The short X.Y version
-version = ''
+version = ""
 # The full version, including alpha/beta/rc tags
-release = ''
+release = ""
 
 
 # -- General configuration ---------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
 #
-# needs_sphinx = '1.0'
+# needs_sphinx = "1.0"
```

Comment:
The theme requires a minimum of 6.2.1, do we want to set this here?

Reply:
Sure, I've set it.